torchtitan/parallelisms/parallelize_llama.py

```diff
@@ -214,12 +214,32 @@ def parallelize_llama(model, world_mesh, parallel_dims, job_config: JobConfig):
             param_dtype=torch.bfloat16, reduce_dtype=torch.float32
         )
         ac_mode = job_config.activation_checkpoint.mode
+        # specifically compile just the RMSNorm layers
+        enable_compile_rmsnorm = job_config.training.compile_rmsnorm
```

Comment:
just calling it `compile_rmsnorm` is good enough I think

Reply:
I think it's better to write `compile_rmsnorm = job_config.model.norm_type == "rmsnorm" and job_config.training.compile_rmsnorm and not job_config.training.compile`